### DISCUSSION TITLE
Welcome to hugo_site Discussions!

### DISCUSSION THREAD

**Discussion by brycewray** [brycewray/hugo-site#1] [Announcements]
Truthfully, I'm adding this only because I am considering — **considering**, mind you — using [giscus](https://giscus.app/), after a reader tipped me onto its existence. Having previously tested its inspiration, [utterances](https://utteranc.es/), I am curious. — BW

**Later**: instead of using this repo’s Discussions for giscus, I am instead doing so in a dedicated “Comments” repo so, if desired, I can build the site with a different repo but point always to the “Comments” repo for giscus purposes.

**Comment by maw246** [comment]
For lack of a better place to post this (I didn't want to authorize Giscus 😬 to make a comment on your blog)... I just finished reading your article about switching from Hugo to Eleventy, since I'm comparing the two for my own project. I have some experience with Eleventy, and just now dabbling in Hugo.  Did you switch your site back to Hugo?  If so, did you ever write up your reasoning in a blog post?  I couldn't find it. But I know it's been a little bit back & forth for you on preferred SSG! 😄   Thanks.

**Reply by brycewray** [reply]
@maw246 Thanks for stopping by! Assuming you mean my September, 2019, post about Hugo -> Eleventy, that’s definitely only a small part of what I ended up calling my “Dance” among multiple SSGs over the last three years. 😄 As of now, I’m on Hugo again but am working on a possible switch either back to Eleventy or to Astro (on which I briefly had the site earlier this year).

**Reply by maw246** [reply]
I hadn't even considered Astro; I figured it was just one more among the countless SSGs out there... and I vaguely recalled reading somewhere that it's slow to build. But after reading your comment, I decided to actually look into it.  Looks impressive!  And it might be exactly the sort of hybrid I need to support some javascript/interactivity on a few "islands" of my site without going full React (a la Gatsby). Thanks for the quick reply!

**Reply by brycewray** [reply]
@maw246 You’re welcome. Also: if you don’t mind getting into a more obscure SSG, you might also take a look at [îles](https://iles.pages.dev/). While it has all the usual limitations of a one-dev project, it’s rather impressive given that it **is** a one-dev project.
